lib/blackjack_rb.rb

```diff
@@ -5,11 +5,18 @@ module BlackjackRb
   require_relative './blackjack_rb/card.rb'
   require_relative './blackjack_rb/dealer.rb'
   require_relative './blackjack_rb/game.rb'
+  require_relative './blackjack_rb/renderer.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.